sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/implementation/FeatureFilterUtils.java

```diff
@@ -3,24 +3,33 @@
 
 package com.azure.spring.cloud.feature.management.implementation;
 
-import org.springframework.util.StringUtils;
-
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Map;
 
+import org.springframework.util.StringUtils;
+
 public class FeatureFilterUtils {
+    
+    public static void updateValueFromMapToList(Map<String, Object> parameters, String key) {
```

Comment:
Should probably add comment. I only say that because I see duplicated comments in other classes. Not sure if there's a way to reuse the comments for overloads like this.